### PR TITLE
hub: update to 2.8.4.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by maxice8** [comment]
- ruby-ronn isn't used anymore
- it now uses md2roff which is compiled in

**Comment by maxice8** [comment]
the building of the md2roff go binary needs to be modified to only build on host

File: srcpkgs/hub/template [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
+	# Required for building md2roff for the host
+	case "$XBPS_MACHINE" in
+		aarch64*) export GOARCH=arm64;;
+		armv6*) export GOARCH=arm; export GOARM=6;;
+		armv7*) export GOARCH=arm; export GOARM=7;;
+		i686*) export GOARCH=386;;
+		x86_64*) export GOARCH=amd64;;
+		ppc64le*) export GOARCH=ppc64le;;
+		ppc64*) export GOARCH=ppc64;;
+	esac
```
Comment (Cogitri) [inline srcpkgs/hub/template]: It'd be prettier if we did something like this in a build_helper or added it to the go build-style (maybe via (XBPS_)HOST_GOARCH?

File: srcpkgs/hub/template [hunk 2]
```diff
@@ -1,44 +1,59 @@
...
 license="MIT"
 homepage="https://hub.github.com/"
 changelog="https://github.com/github/hub/releases"
 distfiles="https://github.com/github/hub/archive/v${version}.tar.gz"
-checksum=83fac8b47b2f56da1e7366688df4bd0eeb5fa20bde5d3de39792ea1cc1d15cd4
+checksum=0aa1342ac5701dc9b7e787ad69640ede06fc84cbe88fb63440b81aca4d4b6273
 
-if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" hub"
+subpackages=" "
```
Comment (maxice8) [inline srcpkgs/hub/template]: remove it

File: srcpkgs/hub/template [hunk 3]
```diff
@@ -1,44 +1,59 @@
...
+
+if ! [ "$CROSS_BUILD" ]; then
```
Comment (maxice8) [inline srcpkgs/hub/template]: we prefer -z "$CROSS_BUILD"

**Comment by maxice8** [comment]
Superseeded by #8749

**Pull request closed without merging**